roles/openshift_hosted/README.md

```diff
@@ -55,6 +63,11 @@ Example Playbook
       cafile: /path/to/my-router-ca.crt
     openshift_hosted_router_registryurl: 'registry.access.redhat.com/openshift3/ose-haproxy-router:v3.0.2.0'
     openshift_hosted_router_selector: 'type=infra'
+    openshift_hosted_registry_storage_kind=glusterfs
+    openshift_hosted_registry_storage_glusterfs_path=external_registry_path
```

Comment:
Change the value to something like `external_glusterfs_volume_name`

Reply:
this one comes from oo_filters.py .lets keep it as it is if possible. Will check how I can reuse it smoothly.

Reply:
I didn't mean the variable name, I meant the example value you're assigning to it.

Reply:
ou :) k